datarobot_batch_scoring/batch_scoring.py

```diff
@@ -70,14 +75,13 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
         network_queue = conc_manager.Queue(queue_size)
         network_deque = conc_manager.Queue(queue_size)
         writer_queue = conc_manager.Queue(queue_size)
-        if not api_token:
-            if not pwd:
-                pwd = ui.getpass()
-            try:
-                api_token = acquire_api_token(base_url, base_headers, user,
-                                              pwd, create_api_token, ui)
-            except Exception as e:
-                ui.fatal(str(e))
+        progress_queue = conc_manager.Queue()
+
+        shovel_status = conc_manager.Value('c', b'-')
```

Comment:
using this is a good idea. By default `lock` is true. If we don't actually need that lock than we should set it false.